components/organisation-unit-tree/src/organisation-unit-node/organisation-unit-node.js

```diff
@@ -52,14 +53,26 @@ export const OrganisationUnitNode = ({
     onCollapse,
     onExpand,
 }) => {
-    const { loading, error, data } = useOrgData(id, {
-        isUserDataViewFallback,
-        suppressAlphabeticalSorting,
-        displayName,
+    const { loading, error, data } = useQuery(fetchOrgData, {
+        defaultData: {
+            organisationUnit: { id, displayName },
+        },
+        initialArguments: [{
+            variables: { id, isUserDataViewFallback },
```

Comment:
Does this need `paging: false`? Some other org unit node queries have it

Reply:
No, I've removed it from the other queries